### ISSUE TITLE
Provide deltarpms for rpm repositories

### ISSUE THREAD

**Issue by widhalmt** [Icinga/icinga-packaging#139]
Hi,

Deltarpms would make updates easier for users in places where broadband internet is not readily available and in especially in remote scenarios where you're working with cellphone connections e.g. during trainings. The same goes for mass installations without local mirror.

Since the tools for building deltarpms are available it should be fairly easy to integrate them into to the repo building process.

Cheers,
Thomas

**Comment by dnsmichi** [comment]
From my experience, also with the Vagrant boxes, deltarpms cause a significant increase in CPU load on the client when rebuilding the full binary from the diff. I've had that enabled in a CentOS 7 VM and provisioning ran into a timeout.

Before having them integrating into the build process, I'd like to see how much time/resources they're consuming on both the build infrastructure and respective clients. 

Also, for snapshot packages with many changes between git commits, they should be disabled.

**Comment by widhalmt** [comment]
The nice thing about deltarpms is that you can choose what's best for your needs. If have low CPU ressources but a thick cable, go without delta, if you have plenty of CPU with just low bandwith use deltarpms.

Deltarpms are built after the whole build process is through so package building won't be affected.

You can always disable them for certain but AFAIR deltarpm first checks if it makes sense to build a delta or if a simple update is better. Prove me wrong @dgoetz if I mixed up things.

**Comment by dgoetz** [comment]
Delta RPMs are not part of build process, but publishing process (just an option of createrepo which creates Delta RPMs if multiple versions of a package are in the same repository). So if it would make no sense for snapshots we could enable/disable it per repo. But especially for snapshot changes between RPMs should be small, resulting in small Delta RPMs. I could run a local test by rsyncing the packages and rerun createrepo (or better createrepo_c) with --deltas if interested.

And yes, it needs CPU and IO to rebuild binaries. But there are several options. If you do not install deltarpm it will not be used at all. Setting deltarpm and deltarpm_percentage in yum.conf/dnf.conf will adjust behavior, by default it will reconstruct 2 packages in parallel and only use delta rpms if not bigger than 75% of original. Setting any of this to 0 will also disable the feature, while -1 for deltarpm will adjust to your number of cores.

**Comment by dgoetz** [comment]
I started my test with rsyncing the epel directory from packages.icinga.com which should be big enough with 4 operating system releases to be representative. 

Initial size is:
```
$ du -hs epel/
14G	epel/
$ du -s epel/
13957012	epel/
```

To create deltarpms for **all** repositories I ran the follow command. It spawns always 5 workers per repository consuming about 50% of one cpu each to create one deltarpm per rpm, so advantages of deltarpms would be available for a direct upgrade only. I had to add option --oldpackagedirs for subdirectories because Icinga repositories are Suse style (one including all archs) and not Red Hat style (one per arch, noarch hardlinked). I shortened the output to include only one repository. (Also note: I used createrepo_c during my tests, createrepo still used on debian could be slower or use more resources)
```
$ date && find epel/{5,6,7,8}/ -mindepth 1 -maxdepth 1 | xargs -I % time createrepo_c --deltas --oldpackagedirs %/x86_64 --oldpackagedirs %/i386 --oldpackagedirs %/i586 --oldpackagedirs %/i686 --oldpackagedirs %/noarch % && date
Mi 31. Jul 14:35:57 CEST 2019
...
Directory walk started
Directory walk done - 565 packages
Temporary output repo path: epel/7/release/.repodata/
Preparing sqlite DBs
Pool started (with 5 workers)
Pool finished
epel/7/release/drpms/icinga2-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingacli-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.4.0-2.el7.centos_2.4.1-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga-idoutils-1.9.3-1.el7.centos_1.13.3-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-selinux-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-selinux-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingabeat-6.3.3-1_6.5.4-1.i686.drpm
epel/7/release/drpms/php-Icinga-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icingabeat-6.1.1-1_6.3.3-1.x86_64.drpm
epel/7/release/drpms/icinga2-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga-idoutils-libdbi-pgsql-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-selinux-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingacli-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/php-Icinga-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-4.8.0-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-selinux-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-common-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/nano-icinga2-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga-idoutils-libdbi-mysql-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icingabeat-5.6.6-1_6.1.1-1.i686.drpm
epel/7/release/drpms/vim-icinga2-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-selinux-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingacli-2.4.0-1.el7.centos_2.4.0-2.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-common-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingacli-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-libs-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingacli-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-selinux-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-0.7.0-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-selinux-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-common-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.4.1-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-libs-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-libs-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga-debuginfo-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-4.7.0-1.el7.centos_4.8.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-common-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga-gui-config-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-selinux-2.4.0-1.el7.centos_2.4.0-2.el7.centos.noarch.drpm
epel/7/release/drpms/icinga-web-module-pnp-1.13.0-1.el7.centos_1.14.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga-gui-1.9.3-1.el7.centos_1.13.3-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga-web-mysql-1.13.0-1.el7.centos_1.14.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-libs-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingacli-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingacli-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-classicui-config-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icingabeat-6.1.1-1_6.3.3-1.i686.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/nano-icinga2-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.4.10-1.el7.centos_2.5.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga-web-1.13.0-1.el7.centos_1.14.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.4.0-2.el7.centos_2.4.1-1.el7.centos.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingabeat-5.6.6-1_6.1.1-1.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-selinux-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-common-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.3.4-1.el7.centos_2.4.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.4.10-1.el7.centos_2.5.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-libs-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-selinux-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingacli-2.4.1-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-2.4.0-1.el7.centos_2.4.0-2.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-common-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-classicui-config-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.4.0-1.el7.centos_2.4.0-2.el7.centos.noarch.drpm
epel/7/release/drpms/icinga-idoutils-libdbi-mysql-1.9.3-1.el7.centos_1.13.3-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingacli-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.4.10-1.el7.centos_2.5.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga-devel-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga-web-pgsql-1.13.0-1.el7.centos_1.14.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/nano-icinga2-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-selinux-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga-rpm-release-7-1.el7.centos_7-2.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-selinux-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-0.4.0-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/nano-icinga2-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-1.12.20-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-selinux-2.4.0-2.el7.centos_2.4.1-1.el7.centos.noarch.drpm
epel/7/release/drpms/icingaweb2-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-common-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga-idoutils-libdbi-pgsql-1.9.3-1.el7.centos_1.13.3-0.el7.centos.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.10.2-1.el7.icinga_2.10.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingacli-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/nano-icinga2-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-common-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga-gui-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-1.0.0-1.el7.centos_1.6.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-1.0.1-1.el7.centos_1.1.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/nano-icinga2-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-selinux-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga-devel-1.9.3-1.el7.centos_1.13.3-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga-debuginfo-1.9.3-1.el7.centos_1.13.3-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/vim-icinga2-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.8.2-1.el7.icinga_2.8.3-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.3.4-1.el7.centos_2.4.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-1.1.0-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-classicui-config-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.3.4-1.el7.centos_2.4.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.10.3-1.el7.icinga_2.10.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-selinux-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga-web-scheduler-1.13.0-1.el7.centos_1.14.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-0.6.2-1.el7.centos_0.7.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-selinux-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-JShrink-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/php-Icinga-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga-doc-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga-idoutils-1.13.3-0.el7.centos_1.14.0-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/php-Icinga-2.4.0-1.el7.centos_2.4.0-2.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.6.3-1.el7.centos_2.7.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.4.10-1.el7.centos_2.5.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.10.0-1.el7.icinga_2.10.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-1.6.0-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-selinux-2.4.1-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-common-2.4.0-2.el7.centos_2.4.1-1.el7.centos.noarch.drpm
epel/7/release/drpms/icinga2-libs-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.7.1-1.el7.icinga_2.7.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.10.4-1.el7.icinga_2.10.5-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.6.3-1.el7.icinga_2.7.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingabeat-6.3.3-1_6.5.4-1.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-common-2.4.2-1.el7.icinga_2.5.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingacli-2.6.1-1.el7.icinga_2.6.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.5.4-1.el7.centos_2.6.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.6.1-1.el7.centos_2.6.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-2.4.1-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/nano-icinga2-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.9.2-1.el7.icinga_2.10.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-selinux-2.6.0-1.el7.centos_2.6.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-bin-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/php-Icinga-2.4.1-1.el7.centos_2.4.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-selinux-2.5.0-3.el7.centos_2.5.1-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-zf1-2.5.2-1.el7.icinga_2.5.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-doc-2.4.10-1.el7.centos_2.5.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.9.0-1.el7.icinga_2.9.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-libs-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/nano-icinga2-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.4.10-1.el7.centos_2.5.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga-1.9.3-1.el7.centos_1.13.3-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.8.3-1.el7.icinga_2.8.4-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-ido-pgsql-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.8.1-1.el7.icinga_2.8.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.4.10-1.el7.centos_2.5.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-lessphp-2.5.1-1.el7.icinga_2.5.2-1.el7.icinga.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.6.2-1.el7.centos_2.6.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.8.4-1.el7.icinga_2.9.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-selinux-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/nano-icinga2-2.10.1-1.el7.icinga_2.10.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-2.7.2-1.el7.icinga_2.8.0-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.6.0-1.el7.icinga_2.6.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.9.1-1.el7.icinga_2.9.2-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-dompdf-2.6.2-1.el7.icinga_2.6.3-1.el7.icinga.noarch.drpm
epel/7/release/drpms/vim-icinga2-2.5.2-1.el7.centos_2.5.3-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga-doc-1.9.3-1.el7.centos_1.13.3-0.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-classicui-config-2.7.0-1.el7.icinga_2.7.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.8.0-1.el7.icinga_2.8.1-1.el7.icinga.x86_64.drpm
epel/7/release/drpms/icinga2-common-2.5.1-1.el7.centos_2.5.2-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-HTMLPurifier-2.5.0-1.el7.icinga_2.5.1-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icinga2-bin-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-doc-2.5.0-1.el7.centos_2.5.0-2.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-ido-mysql-2.5.3-1.el7.centos_2.5.4-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-selinux-2.5.0-2.el7.centos_2.5.0-3.el7.centos.x86_64.drpm
epel/7/release/drpms/icinga2-2.4.10-1.el7.centos_2.5.0-1.el7.centos.x86_64.drpm
epel/7/release/drpms/icingaweb2-vendor-Parsedown-2.5.3-1.el7.icinga_2.6.0-1.el7.icinga.noarch.drpm
epel/7/release/drpms/icingacli-2.3.4-1.el7.centos_2.4.0-1.el7.centos.noarch.drpm
epel/7/release/drpms/icingacli-2.4.0-2.el7.centos_2.4.1-1.el7.centos.noarch.drpm
212.96user 6.12system 0:59.45elapsed 368%CPU (0avgtext+0avgdata 183768maxresident)k
551488inputs+206080outputs (0major+2645339minor)pagefaults 0swaps
...
Mi 31. Jul 14:54:55 CEST 2019
```

So it took 20 minutes to create all the deltarpm for epel with a load average increase by 2.5 and near to no memory consumption. Size has increased slightly (883 MB).
```
$ du -hs epel/
15G	epel/
$ du -s epel/
14862064	epel/
```

If you think it makes no sense to have this for snapshot repositories, because noone will do a direct upgrade there, size could be reduced about 28MB again.

So my personal conclusion would be to run it once to create the deltarpms initially without disturbing the pipeline or something similar (probably only for still supported distributions) and then add them simply during publish process because resource consumption is not very high and it will be an improvement for at least some users.

I am not sure how Suse handles deltarpm. Is support for them also enabled by default? Anything special? Perhaps @carraroj does know it for being an OpenSUSE user.

**Comment by widhalmt** [comment]
I want to emphasize that this test was run on the full repositories. Already created deltarpms will not be touched during new releases which means that this test shows the worst case scenario during a complete rebuild. Consecutive creation will be *much* faster.

**Comment by lazyfrosch** [comment]
Our packaging server is currently build on Debian/Ubuntu, which does all the handling of packages with custom scripts and createrepo.

I'm not even sure if deltarpms are possible there.

At some point we need to refactor most of the packaging server parts, as I did for the build system.

Maybe then, but just maybe...

**Comment by dgoetz** [comment]
Package deltarpm is available for all current Debian and Ubuntu versions, so it should be possible.

**Comment by dnsmichi** [comment]
There's currently no plans to add this, with the other DEV todos in mind.

**Comment by htriem** [comment]
Looking at it - is there still a need for this? Need to know for planning reasons. @dgoetz @widhalmt

**Comment by widhalmt** [comment]
To be honest: There's no *need*. But since it should be easily doable (depending on the way, you're managing the repos) it's a nice-to-have that could help with saving bandwidth.

**Comment by htriem** [comment]
Okay, I will keep this on the agenda then. Thanks for the quick response! 

I _really_ can't make any promises about it though, for now.

**Comment by Al2Klimov** [comment]
First, @dgoetz will figure out how many deltas are reasonable, as we have createrepo now.

**Comment by Al2Klimov** [comment]
Downloaded https://packages.icinga.com/centos/7/release/ and ran createrepo --deltas myself.

|                              | du -sh .
|-----------------|------
| (did nothing at all) | 4.3G
| createrepo . | 4.3G (obviously)
| createrepo --deltas --oldpackagedirs . . | 5.5G

+28%?! I guess the default --num-deltas 1 is >enough!

* In any case #302 should be fixed first.

**Comment by julianbrost** [comment]
> +28%?! I guess the default --num-deltas 1 is >enough!

Isn't the idea of delta rpms to *additionally* provide packages that contain just the differences to save download sizes? An increase in repo size sounds like it is to be expected.

**Comment by Al2Klimov** [comment]
Yes, but --num-deltas 2 makes it +56%(!), so -as I said- I prefer +28%.

**Comment by Al2Klimov** [comment]
'seems the feature itself got dropped in Fedora 40: https://fedoraproject.org/wiki/Changes/Drop_Delta_RPMs
